### PR TITLE
Clippy fixes

### PR DESCRIPTION
**Pull request by XVilka**
- Just some minor code style improvements.
- Updated Rust version to 1.38, because otherwise AppVeyor was failing over `whoami` crate (was 1.33).
- Updated the dependencies

### CODE REVIEW

File: src/ui_model/item.rs [hunk 1]
```diff
@@ -83,7 +83,7 @@ impl InkOverflow {
             right = 0;
         }
 
-        if left == 0 && right == 0 && top == 0 && right == 0 {
```
Comment (daa84) [inline src/ui_model/item.rs]: Hm, looks like here bottom == 0

Comment (XVilka) [reply src/ui_model/item.rs]: Done, thanks!

**Comment by XVilka** [comment]
I fixed the logic error, please check it again.

**Comment by daa84** [comment]
Thanks :+1:

**Pull request merged**